balena/models/device_os.py

```diff
@@ -68,14 +68,21 @@ def get_config(self, app_id, options):
             endpoint=self.settings.get('api_endpoint')
         )
 
-    def download(self, raw=None, **data):
```

Comment:
Was this working w/o arguments before (picking the latest as a default)?

Reply:
version wasn't mandatory and it's parsed properly so this didn't work